src/modules/ModuleGovernance/query.proposal.ts

```diff
@@ -0,0 +1,51 @@
+import { useQuery } from '@vue/apollo-composable'
+import gql from 'graphql-tag'
+import { Ref } from 'vue'
+import { Address } from '@/core/kaikas'
+import { ProposalState } from './types'
+
+export interface ProposalQueryResult {
+  proposal: {
+    id: Address
+    title: string
+    start: number
+    end: number
+    state: ProposalState
+    choices: string[]
+    scores: number[],
+    scores_total: number,
+    body: string
+    author: Address,
+    snapshot: number
+  }
+}
+
+export function useProposalQuery(id: Ref<Address>) {
+  return useQuery<ProposalQueryResult>(
+    gql`
+      query ProposalQuery($id: String!) {
+        proposal(
+          id: $id
+        ) {
+          id
+          title
+          start
+          end
+          state
+          choices
+          scores
+          scores_total
+          body
+          author
+          snapshot
+        }
+      }
+    `,
+    () => ({
+      id: id.value
+    }),
+    {
+      clientId: 'snapshot',
```

Comment:
Hm, I got an idea to define all client names in the root `const` and use it then instead of hardcoding `clientId` everywhere.

Reply:
Yeah, let's do it in another PR